docs/modules/ROOT/pages/usage.adoc

```diff
@@ -308,7 +308,29 @@ spec:
 
 ==== LDAP
 
-Currently not supported.
+Druid supports authentication of users against an LDAP server.
+Have a look at https://github.com/stackabletech/druid-operator/tree/main/tests/templates/kuttl/ldap-authentication[the LDAP test] and the general xref:commons-operator::authenticationclass.adoc[Stackable Authentication] documentation on how to set it up.
+
+In general, it requires you to specify a xref:commons-operator::authenticationclass.adoc[`AuthenticationClass`] which is used to authenticate the users.
+Here you have a reduced, non-functional YAML example, which shows the most essential parts which are needed to configure LDAP in the druid-operator:
+
+[source,yaml]
+----
+include::example$code/druid-ldap-authentication.yaml[]
+----
+
+Refer to https://github.com/stackabletech/druid-operator/tree/main/tests/templates/kuttl/ldap-authentication/create_ldap_user.sh[the LDAP test user creation script] to see what LDAP settings are referred to in this config snippet.
+
+Please note, that Druid makes very specific assumptions about the content of the connected LDAP. Specifically, you will need to create an admin "Group" and assign your chosen admin user, as well as the internal user, to it. See the https://github.com/stackabletech/druid-operator/tree/main/tests/templates/kuttl/ldap-authentication/create_ldap_user.sh[create_ldap_user.sh] file in the current LDAP test case for an example.
+
+==== Known Issues and Upcoming Work
+
+By enabling LDAP, the nodes will try to use LDAP to communicate among themselves. This will interfere or override the TLS authentication method at the current moment.
+This might be fixable by adding TLS to the auth chain.
```

Comment:
Configuring both should do this "automagically".

Reply:
Unfortunately this turned out to be non-straightforward. I'd vote to bring it up as a single issue to decide whether it has a high enough priority to fix, as it's somewhat of an edgecase.